/**:
  ros__parameters:
    mass_drill:
      offset:
        - 263616.4375
        - -29098.896484375
        - 263616.4375
        - 263616.4375
      scale:
        - 246.6250305175781
        - 246.6250305175781
        - 246.6250305175781
        - 246.6250305175781
      alpha: 0.8
      enabled_channels:
        - false
        - true
        - false
        - false
    mass_container:
      offset:
        - 263616.4375
        - 263616.4375
        - 263616.4375
        - 263616.4375
      scale:
        - 451.8432922363281
        - 451.8432922363281
        - 451.8432922363281
        - 451.8432922363281
      alpha: 0.8
      enabled_channels:
        - false
        - true
        - false
        - false
    potentiometer:
      min_voltages:
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
      max_voltages:
        - 3.000000001
        - 3.000000001
        - 3.000000001
        - 3.000000001
      min_angles:
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
      max_angles:
        - 180.000000001
        - 180.000000001
        - 180.000000001
        - 180.000000001
      enabled_channels:
        - true
        - true
        - false
        - false
    servo:
      min_duty:
        - 5.000000001
        - 5.000000001
        - 5.000000001
        - 5.000000001
      max_duty:
        - 10.000000001
        - 10.000000001
        - 10.000000001
        - 10.000000001
      min_angles:
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
      max_angles:
        - 180.000000001
        - 180.000000001
        - 180.000000001
        - 180.000000001
    accel:
      bias:
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
      transform:
        - 1.000000001
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.000000001
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.999999943436137e-09
        - 1.000000001
    gyro:
      bias:
        - 1e-09
        - 1e-09
        - 1e-09
    mag:
      hard_iron:
        - -14.66149139404297
        - 5.658301830291748
        - 9.051597595214844
      soft_iron:
        - 0.8947830200195312
        - 0.0144039997830987
        - -0.0100299995392561
        - 0.0144039997830987
        - 0.8612419962882996
        - -0.01099000032991171
        - -0.0100299995392561
        - -0.01099000032991171
        - 0.8623480200767517